src/api/props.js

```diff
@@ -13,9 +13,7 @@ function get(elem) {
 
 function set(elem, newProps) {
   assign(elem, newProps);
-  if (elem.constructor.render) {
-    elem.constructor[$renderer](elem);
-  }
+  elem.constructor[$renderer](elem);
```

Comment:
In my upcoming PR I actually changed this to:

```js
function set(elem, newProps) {
  assign(elem, newProps);

  // `set()` may be called on a non-skate or non-initialised component, which
  // means we can't be sure that `render()` exists.
  const render = elem.constructor[$renderer];
  if (render) {
    render(elem);
  }
}
```

We should probably add a test to lock in the behaviour that `props` can be called on a non-initialised or non-skate element.